java-squid/src/test/files/se/UnclosedResourcesCheck.java

```diff
@@ -240,6 +248,18 @@ public void correctHandlingOfJarFile(String fileName) {
       closeJar(jar);
     }
   }
+
+  public void getDirectivesFromFile(File aFile) {
+    BufferedReader reader = null;
+    try {
+      reader = new BufferedReader(new FileReader(aFile));
+      reader.read();
+//    } catch(IOException e) { // Causes a problem when decommented!
```

Comment:
So then please remove the comment, I can't know (and neitheir further readers) that you are not the one who wrote this... example should not be taken in extenso in unit test files but stripped down to cases.